h/static/scripts/test/app-controller-test.coffee

```diff
@@ -117,7 +117,11 @@ describe 'AppController', ->
 
   it 'does not show login form for logged in users', ->
     createController()
-    assert.isFalse($scope.dialog.visible)
+    assert.isFalse($scope.accountDialog.visible)
+
+  it 'does not show shareDialog by default', ->
```

Comment:
Try to write these in a natural language descriptive style, rather than referencing the implementation. 'does not show the share dialog at start' or something. I'm totally nitpicking, but `shareDialog` is an object, it can't be shown. The "share dialog" is a thing that can be shown.